descartes/package.xml

```diff
@@ -16,7 +16,7 @@
   <run_depend>descartes_planner</run_depend>
   <run_depend>descartes_trajectory</run_depend>
   <run_depend>descartes_utilities</run_depend>
-
+  <run_depend>descartes_tests</run_depend>
```

Comment:
I was just trying to be diligent, and I have very little knowledge of the Debian release process. I'm happy to drop this line if doing so has no real consequence.